.changeset/seven-socks-perform.md

```diff
@@ -2,23 +2,24 @@
 "@logto/schemas": patch
 ---
 
-## Fix third-party app get /interaction/consent 500 bug
+## Resolve Third-Party App's /interaction/consent Endpoint 500 Error
```

Comment:
does it have a rebase issue?

Reply:
By design. This PR will overwrite the previous changeset update.   It's a better fix, which will align the type definition.